### ISSUE TITLE
[Enhancement]:  Alerting page cannot Accessable

### ISSUE THREAD

**Issue by sivaprasath2004** [ChromeGaming/Squard-line#160]
### Do you want to have the enhancement of existing game ? 😀 Describe yourself..

- In this project `Welcome to the squad game` page `Let's play` button not clicked,but the game will be start .

### Video

[Screencast from 2024-06-21 20-47-11.webm](https://github.com/ChromeGaming/Squard-line/assets/121082414/39c00b4a-f1aa-4887-ac33-01bcbdb9d920)

- more information please see the video :middle_finger:  

### Describe the solution you'd like

- please assign me

### Select program in which you are contributing

GSSoC23

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE]() of this project.

**Comment by sivaprasath2004** [comment]
@Durgesh4993  please assign me.

**Comment by Mukesh751** [comment]
@sivaprasath2004 this has been fixed now

**Comment by sivaprasath2004** [comment]
@Mukesh751 if you not assign me, how I can. Fix this , please reopen the issue and assign me..

**Comment by Mukesh751** [comment]
@sivaprasath2004 someone else had already fixed that issue

**Comment by sivaprasath2004** [comment]
@Mukesh751 yes ok, but you will not assign the contributors how I can work the issues. First you will assign then you ask the questions..

**Comment by Mukesh751** [comment]
@sivaprasath2004 no that's not the right way to do things.
